### PR TITLE
code of conduct first pass

### PR DESCRIPTION
**Pull request by tkellen**
cc @leobalter @gnarf

I need to work on this to replace references to the JSFoundation to be TC39, but before I do that, I want to make sure I've captured the whole of the CoC in this file. Can you confirm?

### CODE REVIEW

**Comment by gnarf** [comment]
There was a section in there about how changes should be made to the code itself.

The process might be different for tc39, but you should make sure you keep the section I think.  It's important to know who makes the rules, and how you can suggest new/better words.

**Comment by leobalter** [comment]
I'd like to see individual files for at least Enforcement and Reporting.

> There was a section in there about how changes should be made to the code itself.

It's also good to have that clear

**Comment by tkellen** [comment]
thanks for the feedback ya'll! we are supposed to present all non-technical agenda items tomorrow. i'm totally beat. i'll be up early tomorrow (pre committee meeting) trying to nail down the best set of revisions i can to get this ready.

**Comment by tkellen** [comment]
Okay, I've pushed up each page separately, then a single commit for edits to each page for review. I think there are a fair few open questions here. Is it our expectation here to discuss the open questions with the TC39 delegation and decide on answers as a group? Should we do more diligence and propose something complete?

**Comment by leobalter** [comment]
> Should we do more diligence and propose something complete?

This is good for a headstart. I don't believe we'll be able to have something final to be adopted in this meeting, and this will give enough time for everyone interested in TC39 to review until the next meeting.

The best strategy, IMHO, is to use this as a candidate and ask for reviewers, nominate them, as many as we find interested in this topic. In the next meeting we will bring a complete version to be approved.

For the reporting committee, this is also something to be discussed in the meeting and find people from the delegates to compose this group. I believe this will be the longest part.

File: Enforcement.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+# Enforcement Manual
+
+This is the enforcement manual followed by TC39's Code of Conduct Committee. It's used when we respond to an issue to make sure we're consistent and fair. It should be considered an internal document, but we're publishing it publicly in the interest of transparency.
 
 ### The Code of Conduct Committee
 
 All responses to reports of conduct violations will be managed by a [Code of Conduct Committee][] ("the committee").
 
-The JS Foundation's Board of Directors ("the board") will establish this committee, comprised of at least three members. One member will be designated chair of the committee and will be responsible for all reports back to the board. The board will review membership on a regular basis.
+The TC39 ??? ("the board") will establish this committee, comprised of at least three members. One member will be designated chair of the committee and will be responsible for all reports back to the board. The board will review membership on a regular basis.
```
Comment (leobalter) [inline Enforcement.md]: _TC39 will establish_

File: Enforcement.md [hunk 2]
```diff
@@ -41,18 +43,16 @@ Possible responses may include:
...
 *   A request for a public or private apology. The chair will deliver this request. The committee may, if it chooses, attach "strings" to this request: for example, the committee may ask a violator to apologize in order to retain his or her membership on a mailing list.
 
 Once a resolution is agreed upon, but before it is enacted, the committee will contact the original reporter and any other affected parties and explain the proposed resolution. The committee will ask if this resolution is acceptable, and must note feedback for the record. However, the committee is not required to act on this feedback.
 
-Finally the committee will make a report for the JS Foundation Board (as well as the project teams in the event of an ongoing resolution, such as a ban).
-
-The committee will never publicly discuss the issue; all public statements will be made by the JS Foundation Board.
+Finally the committee will make a report for the TC39 ???. The committee will never publicly discuss the issue; all public statements will be made by the TC39 ???.
```
Comment (leobalter) [inline Enforcement.md]: > The committee will never publicly discuss the issue

I like this being in a new paragraph and we can even bold this part, it addresses some of the already expressed concerns.

> ... make a report for the TC39 ???

make a report of the TC39 members and respective delegates and the ECMA secretariat (?)

> ... public statements will be made by the TC39 ???

TC39 Chair and/or the ECMA Secretariat?

File: FAQ.md [hunk 3]
```diff
@@ -1,30 +1,32 @@
-This FAQ attempts to address common questions and concerns around the JS Foundation's [Code of Conduct][]. If you still have questions after reading it, please feel free to [contact us][].
+# Frequently Asked Questions
+
+This FAQ attempts to address common questions and concerns around TC39's [Code of Conduct][]. If you still have questions after reading it, please feel free to [contact us][].
 
 ### Why have you adopted a Code of Conduct?
 
-We think the JS Foundation community is awesome. If you're familiar with the JS Foundation community, you'll probably notice that the Code basically matches what we already do. Think of this as documentation: we're taking implicit expectations about behavior and making them explicit.
+We think the TC39 community is awesome. If you're familiar with the TC39 community, you'll probably notice that the Code basically matches what we already do. Think of this as documentation: we're taking implicit expectations about behavior and making them explicit.
```
Comment (leobalter) [inline FAQ.md]: > We think the TC39 community is awesome

s/TC39 community/JS community/

> If you're familiar with the TC39 community, you'll probably notice that the Code basically matches what we already do.

I'm not sure this applies in here. We can rephrase this to: "If you're familiar with the TC39, you'll probably notice we are making progressive efforts to promote and improve diversity and inclusion to match the Code and our high technical experience with standards work".

File: FAQ.md [hunk 4]
```diff
@@ -1,30 +1,32 @@
...
 
-We know that the JS Foundation community is open, friendly, and welcoming. We want to make sure everyone else knows it too.
+We know that the TC39 community is open, friendly, and welcoming. We want to make sure everyone else knows it too.
```
Comment (leobalter) [inline FAQ.md]: TC39 is not exactly open, so `s/TC39 community/JS community` could work here or just remove the open or replace it with transparent.

File: FAQ.md [hunk 5]
```diff
@@ -1,30 +1,32 @@
...
 
 ### What does it mean to "adopt" a Code of Conduct?
 
-For the most part, we don't think it means large changes. We think that the text does a really good job describing the way the JS Foundation community already conducts itself. We expect that most people will simply continue to behave in the awesome way they have for years.
+For the most part, we don't think it means large changes. We think that the text does a really good job describing the way the TC39 community already conducts itself. We expect that most people will simply continue to behave in the awesome way they have for years.
```
Comment (leobalter) [inline FAQ.md]: s/TC39 community/TC39

File: FAQ.md [hunk 6]
```diff
@@ -1,30 +1,32 @@
...
 
-However, we do expect that people will abide by the spirit and words of the CoC when in "official" JS Foundation spaces. This code has been adopted by the JS Foundation and all of its projects. That means that it'll apply both in community spaces _and_ at JS Foundation events.
+However, we do expect that people will abide by the spirit and words of the CoC when in "official" TC39 spaces. This code has been adopted by TC39. That means that it'll apply both in community spaces _and_ at TC39 events.
 
-In practice, this means the various JS Foundation IRC channels (#jquery, #jquery-dev, etc.), bug tracking and code review tools, and "official" JS Foundation events such as sprints. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+In practice, this means the various TC39 IRC channels, bug tracking and code review tools, and "official" TC39 events. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
```
Comment (leobalter) [inline FAQ.md]: s/various/any

File: FAQ.md [hunk 7]
```diff
@@ -1,30 +1,32 @@
...
 
-Real-life events, however, require a bit more care. The JS Foundation wants to be sure that all of its events have policies and procedures in place for handling harassment. It's especially important to us that real-life events take steps to protect the physical and mental security of their participants.
+Real-life events, however, require a bit more care. TC39 wants to be sure that all of its events have policies and procedures in place for handling harassment. It's especially important to us that real-life events take steps to protect the physical and mental security of their participants.
 
-So the JS Foundation will require that all events have some sort of anti-harassment policy in place. The JS Foundation thinks the [Ada Initiative's template][] is pretty good, but we're open to alternatives.
+So TC39 will require that all events have some sort of anti-harassment policy in place. TC39 thinks the [Ada Initiative's template][] is pretty good, but we're open to alternatives.
```
Comment (leobalter) [inline FAQ.md]: > TC39 thinks the [Ada Initiative's template][] is pretty good, but we're open to alternatives.

I believe we should remove this suggestion, saying what TC39 think will create bikeshedding.

File: FAQ.md (hunk 4 above)
Comment (MylesBorins) [inline FAQ.md]: +1 for the TC39 is transparent

**Comment by leobalter** [comment]
I left a few comments above, this is gonna be great

File: Changelog.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
-## Change control process
+# Change Control Process
 
-We're (mostly) programmers, so we'll track changes to the Code of Conduct and associated documents the same way we track changes to code. All changes will be proposed via a pull request to the [js.foundation repository on GitHub][]. Changes will be reviewed by the [conduct committee][] first, and then sent to the JS Foundation, the development teams, and the community for comment. We'll hold a comment period of at least one week, and then each group will vote on the change using its normal process (a board for the JS Foundation and project leads for the dev teams). Approved changes will be merged, published, and noted below.
+We're (mostly) programmers, so we'll track changes to the Code of Conduct and associated documents the same way we track changes to code. All changes will be proposed via a pull request to this repository. Changes will be reviewed by the [conduct committee][] first, and then ??? TBD Approved changes will be merged, published, and noted below.
```
Comment (MylesBorins) [inline Changelog.md]: s/???/committee

File: Committee.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+
+* TBD
+* TBD
+* TBD
+* TBD
+* TBD
+
+You can contact [???][]. For more details please see the [Reporting Guidelines][].
+
+[JS Foundation Code of Conduct]: Conduct.md
```
Comment (MylesBorins) [inline Committee.md]: s/JS Foundation/TC39

**Comment by tkellen** [comment]
Thanks @leobalter! I'll address all of your edits in a few minutes.

File: Conduct.md [hunk 10]
```diff
@@ -1,12 +1,12 @@
-Like the technical community as a whole, the various JS Foundation teams and communities are made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching, and connecting people.
+# TC39 Code of Conduct
 
-Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to founders, mentors and those seeking help and guidance.
+Like the technical community as a whole, TC39 is made up of a mixture of professionals and volunteers from all over the world. Diversity is one of our goals, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to all TC39 delegates and those seeking help and guidance.
```
Comment (MylesBorins) [inline Conduct.md]: should we include something about member guests here as well?

File: FAQ.md (hunk 4 above)
Comment (leobalter) [inline FAQ.md]: _We know that the TC39 is transparent, friendly, and welcoming._

sounds good to me. people might argue about being friendly but I'm being optimistic here.

File: Conduct.md [hunk 11]
```diff
@@ -1,12 +1,12 @@
...
 
 This isn't an exhaustive list of things that you can't do. Rather, take it in the spirit in which it's intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
 
-This Code of Conduct applies to all spaces managed by the JS Foundation and its projects. This includes IRC, the mailing lists, the issue trackers, JS Foundation events, and any other forums created by the project teams which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+This Code of Conduct applies to all spaces managed by TC39. This includes IRC, mailing lists, issue trackers, TC39 events, and any other forums created by TC39 delegates which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
```
Comment (MylesBorins) [inline Conduct.md]: "may affect a person's ability to participate within them" 

we should only state this if it is enforceable

File: Conduct.md (hunk 10 above)
Comment (tkellen) [inline Conduct.md]: Yup!

File: Conduct.md (hunk 11 above)
Comment (tkellen) [inline Conduct.md]: Enforcement is definitely an open question throughout this doc. I'm going to land this and make a note of that in the README.
